src/Filter.js

```diff
@@ -92,13 +95,16 @@ function Filter({
 
   const resetFilter = function () {
     // reset the inputstates to one if multiple exist
-    if (inputStates.length > 1) {
-      setInputStates([{ columnName: [''], logicValue: [], inputValue: [] }])
+    if (inputStates[0].inputValue.length) {
+      setInputStates([
+        { columnName: ['HourUTC'], logicValue: [], inputValue: [] },
```

Comment:
@cotts you are hard coding column name here.

Reply:
Because the default filter is `HourUTC` and when reset filters and there is no default column the filter starts with the wrong input like image below.

![image](https://user-images.githubusercontent.com/11343299/112304664-829b9600-8c7c-11eb-847f-316661f4fef1.png)

This set the default filter and when it's loaded the input turns back to the date picker

Reply:
but `Hour UTC` is a column name which doesn't exist in other datasets

Reply:
In this case, I will undo that but will also need to be fixed when resetting the filters.